src/test/compressor/test_compression_plugin_snappy.cc

```diff
@@ -38,12 +38,17 @@ int main(int argc, char **argv) {
   global_init(NULL, args, CEPH_ENTITY_TYPE_CLIENT, CODE_ENVIRONMENT_UTILITY, 0);
   common_init_finish(g_ceph_context);
 
-  int r = system("mkdir -p .libs/compressor");
+  const char* env = getenv("CEPH_LIB");
+  std::string directory(env ? env : "lib");
+  string mkdir_compressor = "mkdir -p " + directory + "/compressor";
```

Comment:
i would prefer to fail with an error if we don't have a CEPH_LIB. an empty `directory` here can't possibly be correct, and will try to modify /